frame/contracts/README.md

```diff
@@ -135,6 +135,18 @@ to `error` in order to prevent them from spamming the console.
 `--dev`: Use a dev chain spec
 `--tmp`: Use temporary storage for chain data (the chain state is deleted on exit)
 
+## Host function tracing
+
+For contract authors, it can be a helpful debugging tool to see which host functions are called, with which arguments, and what was the result. 
```

Comment:
```suggestion
For contract authors, it can be a helpful debugging tool to see which host functions are called, with which arguments, and what the result was. 
```